syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -46,21 +61,20 @@ def __distribute_shares(workers, shares):
         return shares_map
 
     def reconstruct(self):
-        shares_map = self.get_shares_map()
-        shares = self.__retrieve_shares(shares_map)
+        shares = self.retrieve_shares()
         plain_text_mod = self.__sum_shares(shares)
         plain_text = self.__map_modular_to_real(plain_text_mod)
         return plain_text
 
-    def __retrieve_shares(self, shares_map):
-        pointers = self.__retrieve_pointers(shares_map)
+    def retrieve_shares(self):
```

Comment:
Q: Can we make this ```__retrieve_shares```?

Reply:
I meant to make them public since they are very useful for debugging to see the shares before adding them, it's basically the reconstruction algorithm but you'll have to add the shares manually